### ISSUE TITLE
Unable to detach link in flow

### ISSUE THREAD

**Issue by saidhasyim** [directus/directus#22550]
### Describe the Bug

I'm not sure whether this is a bug or I am missing something. But I cannot detach any link in a flow.  https://www.loom.com/share/fcbb919a84a64cb5a73d17b5b7c9fa59?sid=376b7f41-aff5-4dc2-ab27-1ebd952c8e1f

### To Reproduce

Create a flow, link to an action. Try to detach the link.

### Directus Version

v10.11.0

### Hosting Strategy

Self-Hosted (Docker Image)

**Comment by jonaskohl** [comment]
Yes, happens to me, too. Super annoying, especially in conjunction with #14185

**Comment by AlexGaillard** [comment]
Sorry @saidhasyim, but we were unable to recreate this bug. If you are still having this issue are you able to provide more information? Perhaps you're having issue with a particular flow type, or flow structure, or trigger, etc?

**Comment by AlexGaillard** [comment]
Hey @saidhasyim !

Thank you for taking the time to report this issue. We really appreciate the help in improving Directus. Due to limited resources, we’re unable to address every reported bug and must choose what to work on wisely. Based on the information available at present, this issue cannot be prioritized at this time. Please see our [bug triaging process](https://directus.io/docs/community/reporting-and-support/bug-reporting#bug-triage-process) for more information on how we evaluate and manage our GitHub issue backlog.

We’ll close this issue as a wontfix, but are happy to revisit at a later date. If you would like to open a PR to resolve this issue, you’re more than welcome to do so. Thanks!
